src/devices/things/things.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <!--Disabling default items so samples source won't get build by the main library-->
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="*.cs" />
```

Comment:
Instead of having a separate project for this, I believe this should go under Common instead. So have a src/devices/Common/SensorInterfaces folder where all of these guys live, and sensors that implement them would just add the Compile item to it.

Reply:
when 2 projects use the same interface they cannot exchange it this way since type is in both assemblies and would not unify correctly - do you mean to make it internal for now?

Reply:
That is true as long as we ship different assemblies for different sensors. At least the plan for now, is to only have one IoT.Device.Bindings.dll that has everything includding the interfaces, so they would be able to exchange it.

Reply:
@joperezr that is true for the package but not true for development where there are still multiple assemblies